### PR TITLE
Ruff fix D204-D215

### PR DESCRIPTION
**Pull request by nstarman**
This PR targets the pydocstyle errors D204-D215, which are mostly about whitespace. I special-cased the file "astropy/utils/decorators.py" from the indentation checks because the Examples sections in the docstring decorators contains code examples that were getting de-dented. I believe the rest, sans some manual corrections I did in astropy.cosmology, are ruff's automatic fixes.

As part of pre-commit ruff will now auto-fix these error, as part of CI or locally if the contributor has pre-commit installed as a git hook (https://pre-commit.com/#3-install-the-git-hook-scripts).

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Do the proposed changes actually accomplish desired goals?
- [ ] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
- [ ] Are tests added/updated as required? If so, do they follow the [Astropy testing guidelines](https://docs.astropy.org/en/latest/development/testguide.html)?
- [ ] Are docs added/updated as required? If so, do they follow the [Astropy documentation guidelines](https://docs.astropy.org/en/latest/development/docguide.html#astropy-documentation-rules-and-guidelines)?
- [ ] Is rebase and/or squash necessary? If so, please provide the author with appropriate instructions. Also see ["When to rebase and squash commits"](https://docs.astropy.org/en/latest/development/when_to_rebase.html).
- [ ] Did the CI pass? If no, are the failures related? If you need to run daily and weekly cron jobs as part of the PR, please apply the `Extra CI` label. Codestyle issues can be fixed by the [bot](https://docs.astropy.org/en/latest/development/workflow/development_workflow.html#pre-commit).
- [ ] Is a change log needed? If yes, did the change log check pass? If no, add the `no-changelog-entry-needed` label. If this is a manual backport, use the `skip-changelog-checks` label unless special changelog handling is necessary.
- [ ] Is this a big PR that makes a "What's new?" entry worthwhile and if so, is (1) a "what's new" entry included in this PR and (2) the "whatsnew-needed" label applied?
- [ ] Is a milestone set? Milestone must be set but `astropy-bot` check might be missing; do not let the green checkmark fool you.
- [ ] At the time of adding the milestone, if the milestone set requires a backport to release branch(es), apply the appropriate `backport-X.Y.x` label(s) *before* merge.

### CODE REVIEW

File: astropy/visualization/wcsaxes/frame.py [hunk 1]
```diff
@@ -381,8 +381,13 @@ def update_spines(self):
 
     def _update_patch_path(self):
         """Override path patch to include only the outer ellipse,
+<<<<<<< HEAD
```
Comment (mhvk) [inline astropy/visualization/wcsaxes/frame.py:384]: OOPS!!

File: pyproject.toml [hunk 2]
```diff
@@ -280,6 +272,7 @@ exclude=[
 "__init__.py" = ["E402", "F401", "F403"]
 "astropy/io/registry/compat.py" = ["F822"]
 "astropy/modeling/models.py" = ["F401", "F403", "F405"]
+"astropy/utils/decorators.py" = ["D214", "D215"]  # keep Examples section indented.
```
Comment (mhvk) [inline pyproject.toml:275]: How about making those `Examples` proper docstring ones (i.e., add `>>>` before, etc.? Or is the ones with `>>>` already?

**Review by mhvk — COMMENTED** [review]
This looks good (minus the OOPS). I think in most cases these were just long one-liners that were broken up at some point because the line was too long (perhaps even by `black`...).

Would be nice if we didn't have to make an exception for `decorators.py`; query in-line.

File: astropy/visualization/wcsaxes/frame.py (hunk 1 above)
Comment (nstarman) [reply astropy/visualization/wcsaxes/frame.py:384]: Shoot! I'll fix that 😆

Comment (nstarman) [reply astropy/visualization/wcsaxes/frame.py]: Done!

File: pyproject.toml (hunk 2 above)
Comment (nstarman) [reply pyproject.toml:275]: Yeah, the problem is lines like this

```
        >>> help(add) # doctest: +SKIP
        Help on function add in module __main__:
        <BLANKLINE>
        add(num1, num2)
            Perform addition.
            Parameters
            ----------
            num1, num2 : Numbers
            Returns
            -------
            result : Number
```

The Parameters & Returns both want to be dedented.

Comment (mhvk) [reply pyproject.toml:275]: OK, maybe easiest to just skip it for now indeed.

**Review by mhvk — APPROVED** [review]
Looks all good to me -- not sure we really need everybody to look at this individually, the changes are really trivial and obvious.

**Pull request merged**